docs/content/rules/sort-array-includes.mdx

```diff
@@ -129,6 +129,7 @@ Specifies the sorting method.
 - `'natural'` — Sort items in a [natural](https://github.com/yobacca/natural-orderby) order (e.g., “item2” < “item10”).
 - `'line-length'` — Sort items by the length of the code line (shorter lines first).
 - `'custom'` — Sort items using the alphabet entered in the [`alphabet`](#alphabet) option.
+- `'unsorted'` — Do not sort items. To be used with the [`useConfigurationIf`](#useConfigurationIf) option.
```

Comment:
What do you think of the name `'no-sort'` or `'preserve-order'`? Or do you prefer `'unsorted'`?

Reply:
@azat-io 

- I prefer `unsorted` and `no-sort` to `preserve-order` (I feel the first two are more explicit).
- `unsorted` is already used in `customGroups`.
https://github.com/azat-io/eslint-plugin-perfectionist/blob/7fdadd7dcdf96fc259973fad79071d54ceac060f/rules/sort-classes.types.ts#L205-L216

I think it's better for consistency to keep the same name, so I would personally pick `unsorted`.

Reply:
Okay, thank you 👍